.travis.yml

```diff
@@ -9,12 +9,20 @@ cache: pip
 python:
 - 3.7
 
+os: linux
+
 install:
+# Python
 - pip install -r monkey/monkey_island/requirements.txt  # for unit tests
 - pip install flake8 pytest dlint  # for next stages
 - pip install -r monkey/infection_monkey/requirements_linux.txt  # for unit tests
 
 before_script:
+# Set the server config to `testing`, for the UTs to use mongomaock and pass.
```

Comment:
typo, but dont fix if it's the only problem

Reply:
ad1e85f8